web/pipeline/views/general.py

```diff
@@ -37,24 +38,12 @@ class CommunityList(generics.ListAPIView):
 
 
 class ServiceList(generics.ListAPIView):
+    serializer_class = ServiceListSerializer
 
-    queryset = Service.objects.all()
-
-    def get(self, request, format=None):
-        service_values = (
-            Service.objects.filter(hex__community__isnull=False)
-            .prefetch_related('hex__community')
-            .select_related("isp__name")
-            .values('hex__community', 'isp__name', 'technology')
-        )
-        output = []
-        for service in service_values:
-            output.append({
-                'tech': service['technology'],
-                'community': service['hex__community'],
-                'isp': service['isp__name']
-            })
-        return HttpResponse(json.dumps(output), content_type="application/json")
+    def get_queryset(self):
```

Comment:
it's more standard (in DRF) to override `get_queryset()` and use the `serializer_class` property than manually handle it yourself